.github/ISSUE_TEMPLATE/introduce-sgid-dataset.md

```diff
@@ -0,0 +1,78 @@
+---
+name: Introduce data to the SGID
+about: Creation Checklist
+title: Add <dataset name> from <place>
+labels: 'introduction'
+assignees: ''
+---
+
+<!-- 
+Introduce your data, where it comes from, why it is being added etc
+-->
+# Summary
+
+We received yada yada from blahbidy blah. Foo Bar is the data contact and it will be updated quarterly. I propse we call it Baz and put it in this ISO category or group.
```

Comment:
```suggestion
We received yada yada from blahbidy blah. Foo Bar is the data contact and it will be updated quarterly. I propose we call it Baz and put it in this ISO category or group.
```